openhtf/exe/__init__.py

```diff
@@ -148,6 +148,10 @@ def _ThreadProc(self):
         self._test_state = self._MakeTestState(dut_id, suppressor)
         executor = self._MakePhaseExecutor(exit_stack, suppressor)
 
+        if self._teardown_function:
```

Comment:
Add a comment along these lines:

`# Run teardown function as part of test teardown, after the test and before plugs.`